src/modules/avformat/producer_avformat.c

```diff
@@ -2792,6 +2830,27 @@ static int video_codec_init(producer_avformat self, int index, mlt_properties pr
     return self->video_index > -1;
 }
 
+static int pick_video_stream(producer_avformat self, int absolute_index)
```

Comment:
The absolute_index parameter for this function is not necessary. The value passed in by the caller is never used. It will be overwritten by the IF or the ELSE  before it is used.

Reply:
Correct, an artifact of refactoring and debugging. Changed.